andela/ah-bird-box-frontend
# #163388209 Setup Travis, code climate and test environment

**What does this PR do?**
Setups Travis, code climate and test environment for the application. Includes readme badges.

**Description of Task to be completed?**
1. Update .travis.yml file to include to configure Travis and code climate
2. Add the repo to Travis and code climate
3. Configure the test and include sample test

**How should this be manually tested?**
1. After making a push to the repo, observe the readme badges for code climate and Travis
2. Additionally, check the repo statistics at https://codeclimate.com/github/andela/ah-bird-box-frontend
For test:
1. npm install to add dependencies
2. At the terminal run npm run test
3. Type a to run all tests

What are the relevant pivotal tracker stories?
[#163388209](https://www.pivotaltracker.com/story/show/163388209)

Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29451552/53805157-6d028900-3f5a-11e9-9202-20d84084c8d0.png)
